servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/StacklessClosedChannelException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.netty.internal;
+
+import io.servicetalk.concurrent.internal.ThrowableUtils;
+
+import java.nio.channels.ClosedChannelException;
+
+public final class StacklessClosedChannelException extends ClosedChannelException {
```

Comment:
Interesting that the quality task did not complain of the lack of javadoc in a public class \cc @idelpivnitskiy 

Can you add javadocs here and for `newInstance()` method?

Reply:
done... not resolving this so @idelpivnitskiy can still see it tho...

Reply:
Interesting, checkstyle introduced new improved rules for javadoc checks:
MissingJavadocMethod
MissingJavadocPackage
MissingJavadocType
See https://checkstyle.sourceforge.io/config_javadoc.html#MissingJavadocMethod

The old rules (JavadocMethod, JavadocPackage, JavadocType) work as regex, when new rules work on checkstyle token parsing. And it looks like the old rules do not work correctly anymore. I tried locally this PR, when I activate `MissingJavadocMethod` rule, it captures this class (for the previous commit).